### DISCUSSION TITLE
error install W7 Client

### DISCUSSION THREAD

**Discussion by jhonus-code** [immense/Remotely#322] [Q&A]
**What's going on? why not download?**

`2021-04-28 10:49:05.396	[Info]	Install started.
2021-04-28 10:49:09.549	[Error]	The request was aborted: Could not create SSL/TLS secure channel.	   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Net.WebClient.GetWebResponse(WebRequest request, IAsyncResult result)
   at System.Net.WebClient.DownloadBitsResponseCallback(IAsyncResult result)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Remotely.Agent.Installer.Win.Services.InstallerService.<DownloadRemotelyAgent>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Remotely.Agent.Installer.Win.Services.InstallerService.<Install>d__13.MoveNext()	System
2021-04-28 10:56:20.898	[Info]	ServerUrl or OrganizationID param is missing.  Unable to install.`

**Comment by bitbound** [comment]
Hello.

Please see the following thread.  It's probably the same issue:  https://www.reddit.com/r/remotely_app/comments/lsuxnl/error_to_install_the_resident_agent_on_windows_7/

Also note that last line:
`[Info] ServerUrl or OrganizationID param is missing. Unable to install.`

Both the server URL and organization ID need to be valid for the installer to work.

**Reply by jhonus-code** [reply]
None of the answers helped me to solve it, I perfectly meet the requirements.
Windows 7 Service Pack with 1.
And the data that I put is logically correct or at least it works on a W10.

**Reply by bitbound** [reply]
There's more to it than just Win7 SP1.  TLS 1.2 needs to be the default security protocol in WinHTTP.

That said, Windows 7 is passed end of life, and I'll be dropping official support for it soon.  Well, there isn't really any "official support."  This is just a hobby project.  :D

**Comment by jhonus-code** [comment]
Already solved.
It is fixed by checking this.
https://www.ryadel.com/en/enable-tls-1-1-1-2-windows-7-8-os-regedit-patch-download/

**Comment by alencarwell** [comment]
hi, my machines is windows 7 and 8, but this instalation new version is script powershell, so instalations fall....

**Reply by bitbound** [reply]
.NET 7 is end-of-life in May.  In preparation, Remotely was upgraded to .NET 8, which no longer supports Windows 7/8.

**Reply by alexpi4** [reply]
Anyone help? This only happens in Windows 8.1 with .Net4.8 , the installation script "Install-Remotely.ps1" gives an error on one line (127) and does not complete the installation. Furthermore, I can't find the "Remotely_Installer.exe" file to try to install manually.

**Comment by alexpi4** [comment]
Tip on how to download the Installer by "nittanygeek"

access your server by your installation address + port

Example

"http://192.168.1.211:5000/Content/Remotely_Installer.exe" and download

the executable.

Thanks for the Tip "nittanygeek"

After downloading the executable, follow the steps below

Installing manually on WINDOWS 8.1
If necessary, install [dot.net](http://dot.net/) 4.8, on some machines it may be necessary.

open CMD as administrator

copy "Remotely_Installer.exe" and "Remotely-Win-x64.zip" to c:\

open cmd as administrator

go to c:\

copy and paste the command below using the mouse button and include your "organizationid"

Remotely_Installer.exe -install -organizationid "xxxxx-xxxxx-xxxxx-xxxx" -serverurl "Youhost:port" -path "c:\Remotely-Win-x64.zip"

hit enter

and wait for the installation

:)

ALTERNATIVE COMMAND
Remotely_Installer.exe -install -organizationid "xxxxx-xxxxx-xxxxx-xxxx" -serverurl "http://192.168.1.211:5000/"
